qase-javascript-commons/src/reporters/testops-reporter.ts

```diff
@@ -227,6 +227,7 @@ export class TestOpsReporter extends AbstractReporter {
     }
 
     this.run.id = result.id;
+    process.env['QASE_TESTOPS_RUN_ID'] = String(result.id);
```

Comment:
Will other threads wait for this to complete before they read the configuration?

Reply:
Oh, and do they share the environment with the controller thread?